cornice/util.py

```diff
@@ -128,7 +128,13 @@ def extract_json_data(request):
     if request.body:
         try:
             body = simplejson.loads(request.body)
-            return body
+            if isinstance(body, dict):
```

Comment:
Actually schema does inherit from ``CorniceSchema`` I added an assertion to make sure it works.